config/groups.yaml

```diff
@@ -18,7 +18,7 @@
     - "2616477"
     - "2707500"
   id: 189398
-- name: "Untitled group"
+- name: "SSH - Assigned Group 3"
```

Comment:
Are you _sure_ that's the name you want?

Reply:
I was going to write a better one when we decided on our edr.